README.md

```diff
@@ -63,178 +72,342 @@ bambuAnnotations <- prepareAnnotations(gtf.file)
+bambu(reads = test.bam, annotations = annotations, genome = fa.file)
+```
+### Running multiple samples
 
-For very large samples (>100 million reads) where memory is limiting we recommend running Bambu in lowMemory mode:
+If you have multiple replicates for a sample, or plan to do comparative analysis between conditions, it may be beneficial to run all samples together instead of individually. This can be done by providing a vector of paths to all the bam files you want to analyze together.
 
 ```rscript
-bambu(reads = test.bam, annotations = bambuAnnotations, genome = fa.file, lowMemory = TRUE)
-```
 
----
+se.multiSample = bambu(reads = c(test1.bam, test2.bam, test3.bam), annotations = gtf.file, genome = fa.file)
+```
 
+The advantage of running samples together include:
+Novel transcripts that are identified in multiple samples are assigned unified IDs, enabling comparative analysis between different samples. This is especially important for downstream differential expression analysis when looking at novel transcripts. Running multiple samples can be multithreaded (see ncore). While running multiple samples, By default, Bambu will train a model separately on each sample and score novel transcripts in each sample separately.
 
-### Use precalculated annotation objects
+If you need to combine samples in multiple configurations (for example different pairwise comparisons) we would recommend using the intermediate rcFiles to save processing time (see [Storing and using preprocessed files (rcFiles)](#Storing-and-using-preprocessed-files-(rcFiles)))
+ 
+### Modulating the sensitivity of discovery (pre and post analysis)
 
-You can also use precalculated annotations.
+When doing transcript discovery there is a balance between sensitivity (the number of real novel transcripts that are detected) and the precision (how many of the novel transcripts are real). To control this balance bambu uses the novel discovery rate (NDR) as the main parameter. The NDR threshold approximates the proportion of novel candidates output by bambu, relative to the number of known transcripts it found. I.E an NDR of 0.1 would mean that 10% of all transcripts passing the threshold are classified as novel. 
 
-If you plan to run ***bambu*** more frequently, we recommend to save the bambuAnnotations object.
+If you are using a genome where you expect a high amount of novel transcripts, a higher NDR is recommended so that these novel transcripts are not missed. Conversely if you are using a well annotated genome, we recommend a lower NDR threshold to reduce the presence of false positives. By default the NDR threshold is automatically chosen for the user based on predicted level of annotation completeness when compared to the default model trained on human reference annotations (Hg38). For more information see [Training a model on another species/dataset and applying it](#Training-a-model-on-another-species/dataset-and-applying-it)
```

Comment:
```suggestion
If you are using a genome where you expect a high amount of novel transcripts, a higher NDR is recommended so that these novel transcripts are not missed. Conversely if you are using a well annotated genome, we recommend a lower NDR threshold to reduce the presence of false positives. By default the NDR threshold is automatically chosen for the user based on predicted level of annotation completeness when compared to the default model trained on human reference annotations (Hg38). For more information see [Training a model on another species/dataset and applying it](#Training-a-model-on-another-speciesdataset-and-applying-it)
```